packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/useDeleteView.ts

```diff
@@ -16,12 +16,13 @@ export const useDeleteView = ({
 }): DeleteViewState => {
   const [
     {
-      history,
+      location,
       locationItems: { fileDataItems: selected },
     },
     dispatchStoreAction,
   ] = useStore();
-  const { current } = history;
+  const { current, path = '' } = location;
```

Comment:
Might require deeper thought on the default values of `location.current`, but wondering if `path` should default to an empty string in the `LocationProvider` logic? Feels somewhat fragile from a maintenance perspective to fallback to a default value in consumers, would also reduce the amount of branches for unit testing